### PR TITLE
Spanish Translation

### PR DESCRIPTION
**Pull request by Konata400**

### CODE REVIEW

File: Langs/1034.lang [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+64 Bit CPU=64 Bit CPU
+64 Bit OS=64 Bit SO
+32 Bit CPU=32 Bit CPU
+32 Bit OS=32 Bit SO
+Unable to Determine=Incapaz de determinar
+Unable to Check List=Incapaz de comprobar listado
+Error Accessing List=Error acceder listado
+Not Currently Listed as Compatible=Actualmente no figura como compatible
+Listed as Compatible=Listado como compatible
+Cores=Nucleos
```
Comment (micwoj92) [inline Langs/1034.lang:33]: Shouldn't it be "núcleos"?

File: Langs/1034.lang [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+Disabled / Not Detected=Desactivado / no detectado
+Drive(s) Meet Requirements=La unidad/es cumplen con los requisitos
+TPM Missing / Disabled=Falta TPM / Desactivado
+Detected=Detectado
+DirectX 12 and WDDM 3=DirectX 12 y WDDM 3
+DirectX 12 and WDDM 2=DirectX 12 y WDDM 2
+No DirectX 12, but WDDM2=Sin DirectX 12, pero WDDM2
+DirectX 12, but no WDDM2=DirectX 12, pero sin WDDM2
+No DirectX 12 or WDDM2=No DirectX 12 o WDDM2
+Test Build?=Copilacion Test?
```
Comment (micwoj92) [inline Langs/1034.lang:49]: I think here's also a typo.

**Review by micwoj92 — CHANGES_REQUESTED** [review]
Please check and add all accent marks where missing.

File: Langs/1034.lang (hunk 1 above)
Comment (Konata400) [reply Langs/1034.lang:33]: Yes Núcleos (sorry)

File: Langs/1034.lang (hunk 2 above)
Comment (Konata400) [reply Langs/1034.lang:49]: No DirectX 12, but WDDM2=Sin DirectX 12, pero con WDDM2
DirectX 12, but no WDDM2=DirectX 12, pero sin WDDM2
No DirectX 12 or WDDM2=Sin DirectX 12 o WDDM2
Test Build?=¿Test construcción?

Comment (micwoj92) [reply Langs/1034.lang:49]: You can change your file directly here https://github.com/Konata400/WhyNotWin11/blob/patch-1/Langs/1034.lang

Comment (Konata400) [reply Langs/1034.lang]: Updated

**Comment by micwoj92** [comment]
Thank you for your translation!

**Pull request merged**